client/verta/verta/_internal_utils/_utils.py

```diff
@@ -260,15 +260,25 @@ def email(self):
         else:
             return None
 
-    def _get_visible_orgs(self):
+    def _get_visible_workspaces(self):
         response = self.make_proto_request(
             "GET", "/api/v1/uac-proxy/workspace/getVisibleWorkspaces"
         )
         response = self.must_proto_response(response, Workspace_pb2.Workspaces)
 
-        org_names = map(lambda workspace: workspace.org_name, response.workspace)
-        org_names = filter(None, org_names)
-        return list(org_names)
+        workspace_names = map(lambda workspace: "{}:{}".format(workspace.org_id, workspace.org_name), response.workspace)
```

Comment:
fyi @evgeniilezhnin, I had been talking with Elizabeth about changing UAC so that the client (and client users) would not need to worry about `org_id`, and instead only use workspace name.

Reply:
yes, a PR for that is merged (https://vertaai.atlassian.net/browse/VUMM-808) so reverted this change